### PR TITLE
Debenqify code

### PR DESCRIPTION
**Pull request by jeffarjeffar**
> avoid using additional macros beyond the ones listed above. In particular, for loop macros are highly discouraged.
- USACO Guide, 2020

### CODE REVIEW

**Review by HarryWangATX — CHANGES_REQUESTED** [review]
follow c++ template in contributing camp

**Comment by maggie-j-liu** [comment]
I don't think using the template is necessary... The template represents the macros you can use, but it is not required.

**Comment by thecodingwizard** [comment]
Though using `<bits/stdc++.h>` is highly preferred :)

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]
Use tabs instead of spaces to indent.

**Comment by jeffarjeffar** [comment]
> Use tabs instead of spaces to indent.

Oops didn't notice

**Comment by frodakcin** [comment]
> Though using <bits/stdc++.h> is highly preferred :)

naaaaaaaaahhhh it's fine right? -- as long as there's not too many unnecessary headers

**Comment by thecodingwizard** [comment]
I'd prefer bits, but not particularly strongly. If you think it's fine then it's fine by me :)

**Comment by frodakcin** [comment]
Like some other things (like IO), I think we shouldn't have a preference for either -- the author can choose.

File: solutions/CSES Graph Girth.mdx [hunk 1]
```diff
@@ -26,234 +26,70 @@ This algorithm runs in $\mathcal{O}(N + M)$ time. Since $N$ and $M$ are so small
...
-		for (const auto& x: v) {
-			if (!fst) res += " ";
-			fst = 0; res += ts(x);
+		for (int adj_node : adj[node]) {
+			if (dist[adj_node] == -1) {
+				dist[adj_node] = dist[node] + 1;
+				bfs.push(adj_node);
+			}
+			else {
+				if (dist[adj_node] >= dist[node]) {
```
Comment (maggie-j-liu) [inline solutions/CSES Graph Girth.mdx:62]: change this to else if

File: solutions/CSES Graph Girth.mdx [hunk 2]
```diff
@@ -26,234 +26,70 @@ This algorithm runs in $\mathcal{O}(N + M)$ time. Since $N$ and $M$ are so small
...
+		res = min(res, cycle_len(i));
 	}
-	ps(ans == MOD ? -1 : ans);
+
+	if (res == inf) {
+		cout << -1 << endl;
+		return 0;
+	}
+
+	cout << res << endl;
```
Comment (maggie-j-liu) [inline solutions/CSES Graph Girth.mdx:92]: the ternary expression was shorter... this is okay, up to you whether to change it or not

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

**Review by maggie-j-liu — APPROVED** [review]

File: solutions/CSES Graph Girth.mdx [hunk 3]
```diff
@@ -26,234 +26,68 @@ This algorithm runs in $\mathcal{O}(N + M)$ time. Since $N$ and $M$ are so small
...
-#define ub upper_bound 
-tcT> int lwb(V<T>& a, const T& b) { return int(lb(all(a),b)-bg(a)); }
-
-// loops
-#define FOR(i,a,b) for (int i = (a); i < (b); ++i)
-#define F0R(i,a) FOR(i,0,a)
-#define ROF(i,a,b) for (int i = (b)-1; i >= (a); --i)
-#define R0F(i,a) ROF(i,0,a)
-#define trav(a,x) for (auto& a: x)
+#include <iostream>
```
Comment (jessechoe10) [inline solutions/CSES Graph Girth.mdx:29]: just change everything to `#include <bits/stdc++.h>` - tbh idt we should detemplify that...

**Review by jessechoe10 — CHANGES_REQUESTED** [review]

File: solutions/CSES Graph Girth.mdx (hunk 2 above)
Comment (jessechoe10) [reply solutions/CSES Graph Girth.mdx:90]: Idt it matters lol...

Comment (HarryWangATX) [reply solutions/CSES Graph Girth.mdx:90]: lol i like ternary expressions as well, but its up to u

File: solutions/CSES Graph Girth.mdx (hunk 3 above)
Comment (maggie-j-liu) [reply solutions/CSES Graph Girth.mdx:29]: I think this is fine... up to you whether to change it or not

Comment (thecodingwizard) [reply solutions/CSES Graph Girth.mdx:29]: > naaaaaaaaahhhh it's fine right? -- as long as there's not too many unnecessary headers

(see Frod's comment above. Unless enough people prefer bits/stdc++.h, what is currently here is probably fine)

**Review by jessechoe10 — APPROVED** [review]

**Review by frodakcin — APPROVED** [review]

**Pull request merged**